src/components/helpers.js

```diff
@@ -0,0 +1,10 @@
+import { v4 as uuid } from "uuid";
+
+export const generateCards = (isGameLong) =>
+  [...Array(isGameLong ? 8 : 4).keys()]
```

Comment:
Try to simplify it somehow, for instance `new Array(isGameLong ? 8 : 4).fill.map(...) ...`